logos/src/logos/model_string_parser.py

```
@@ -0,0 +1,86 @@
+                    policy[k] = v
+            else:
+                k, v = param.split("=", maxsplit=1)
+                extra[k] = v
+    except ValueError as e:
+        raise SyntaxError("Invalid logos model string") from e
+    if policy["default"] == "true":
+        policy["default"] = True
+    elif policy["default"] == "false":
+        # Throw error if we don't want only default values but provided nothing else
+        if len(policy) == 1:
+            raise AttributeError("Non-default policy without additional attributes is not allowed. Provide at least one policy attribute to override")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)